### PR TITLE
feat: add simple log action

### PR DESCRIPTION
**Pull request by wolfenrain**
## Status

**READY**

## Description

Add a simple log action, which will be included in newly created `fluttium.yaml` files.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: actions/log_action/lib/src/log_action.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/// {@template log_action}
+/// A simple log action for Fluttium.
+///
+/// An action is immutable and it represents a step in the user flow.
+///
+/// This action can be invoked either using the short-hand version:
+///
+/// ```yaml
+/// - logAction: "Hello World"
```
Comment (felangel) [inline actions/log_action/lib/src/log_action.dart:11]:
```suggestion
/// - log: "Hello World"
```
isn't it `log`?

Comment (wolfenrain) [reply actions/log_action/lib/src/log_action.dart]: Yes! Fixed it

File: actions/log_action/pubspec_overrides.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+dependency_overrides:
+  fluttium:
+    path: /Users/wolfen/Projects/personal/fluttium/packages/fluttium
```
Comment (felangel) [inline actions/log_action/pubspec_overrides.yaml:3]: Do you want to track these files?

Comment (wolfenrain) [reply actions/log_action/pubspec_overrides.yaml:3]: oh this shouldn't be absolute, should be relative. But they should be tracked to make contributing easier. They are ignored by pub publish afaik

**Review by felangel — APPROVED** [review]

**Pull request merged**